agenta-backend/agenta_backend/tasks/evaluations.py

```diff
@@ -121,12 +126,13 @@ async def aggregate_evaluator_results(
     app: AppDB, evaluators_aggregated_data: dict
 ) -> List[AggregatedResult]:
     aggregated_results = []
-    for evaluator_key, values in evaluators_aggregated_data.items():
-        average_value = sum(values) / len(values) if values else 0
-        evaluator_config = await fetch_evaluator_config_by_appId(app.id, evaluator_key)
-        aggregated_result = AggregatedResult(
-            evaluator_config=evaluator_config.id,
-            result=Result(type="number", value=average_value),
-        )
-        aggregated_results.append(aggregated_result)
+    # TODO: find a good solution for aggregating evaluator results
```

Comment:
What is the problem here? 

If the issue is computing averages... Best practices is to use pandas or numpy.

Reply:
there is no problem. we will just work on once all evaluators are done